.github/workflows/release.yml

```
@@ -235,6 +235,11 @@ jobs:
     name: GitHub Release
     needs: build
     runs-on: ubuntu-16.04
+    env:
+      npm-dir: ./installers/npm
+      node-version: '14.x'
+      registry-url: 'https://registry.npmjs.org'
+      is-prerelease: ${{ contains(steps.get_version.outputs.VERSION, 'alpha') || contains(steps.get_version.ouputs.VERSION, 'beta') || contains(steps.get_version.outputs.VERSION, 'test')}}
```

This should probably have `rc` as well, since we tend to use that one the most.
```suggestion
      is-prerelease: ${{ contains(steps.get_version.outputs.VERSION, 'alpha') || contains(steps.get_version.ouputs.VERSION, 'beta') || contains(steps.get_version.ouputs.VERSION, 'rc')|| contains(steps.get_version.outputs.VERSION, 'test')}}
```